test/e2e/testing-manifests/storage-csi/gce-pd/controller.yaml

```diff
@@ -25,6 +25,8 @@ spec:
           args:
             - "--v=5"
             - "--csi-address=/csi/csi.sock"
+            - "--leader-election"
+            - "--leader-election-namespace=$(PDCSI_NAMESPACE)"
```

Comment:
Can namespace be set with downward API instead? Then we don't need to patch it in the manifest

Reply:
Good point.

I guess the pd csi driver yaml (from which this was copied) could be updated too

Reply:
Oh, the pd csi driver was using the downardAPI. Hunh, I thought $(VAR) was only for make.

Reply:
Arg, the parens are because variables in args aren't evaluated in a bash-like context but some custom thing. Now I see. So many ways to step off the path.